docs/content/en/docs/tutorials/tetragon-metrics.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Tetragon Metrics"
+weight: 1
+icon: "overview"
+description: "Fetching and understanding Tetragon metrics"
+---
+
+# Fetching Tetragon Metrics
+
+## Kubernetes
+
+Tetragon, when installed with Helm as mentioned in [Deploying on Kubernetes] ({{< ref "docs/getting-started/deployment/kubernetes" >}}), creates a pod with `tetragon-` prefix. It also creates a service named `tetragon` that exposes metrics on the specified port.
+
+### Getting metrics port
+
+Check if `tetragon` service is running
```

Comment:
```suggestion
Check if the `tetragon` service exists:
```
Small suggestion, services are not really "running".

Reply:
Thank you, Fixed !